apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/src/main/java/com/expediagroup/apiary/extensions/events/metastore/messaging/KafkaMessageTask.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.events.metastore.messaging;
+
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+
+import com.expediagroup.apiary.extensions.events.metastore.common.messaging.Message;
+import com.expediagroup.apiary.extensions.events.metastore.common.messaging.MessageTask;
+
+class KafkaMessageTask implements MessageTask {
```

Comment:
Same here, I think the "common" package should only really contain the event and related classes. If there is any code currently in "common" that is only being sued the the listener then they should be moved into the listener package. The original scope of "common" was far wider than what I think we should be using it for here.